packages/side-nav/src/CollapseButton/CollapseButton.js

```diff
@@ -16,12 +21,14 @@ export default class CollapseButton extends Component {
   };
 
   static defaultProps = {
-    onClick: () => {},
     minimized: false
   };
 
+  createEventHandlers = memoizeCreateButtonEventHandlers();
+
   render() {
     const { minimized, onClick } = this.props;
+    const { handleClick, handleKeyDown } = this.createEventHandlers(onClick);
```

Comment:
I foresee us creating a behavior component to help reduce some of the boilerplate here, but I'd like to make those changes in a separate PR to avoid creating & releasing the `@hig/behaviors` package in this PR.